react/utils/client/fetch.ts

```diff
@@ -0,0 +1,21 @@
+import {format, parse} from 'url'
+
+export const IOFetch = async (uri: string | Request, init?: RequestInit): Promise<Response> => {
+  const {useGetInBatch = false, body} = init || {} as any
+  const formattedInit = {...init}
+  let formattedUri = uri
+
+  if (useGetInBatch && typeof uri === 'string' && formattedInit) {
```

Comment:
Check for Request: https://github.com/vtex/render-server/blob/master/node/middleware/render/sandbox.ts#L24